services/movie_session.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
+
+from datetime import datetime
+from typing import Optional
+from django.db.models import QuerySet
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    movie = Movie.objects.get(pk=movie_id)
+    cinema_hall = CinemaHall.objects.get(pk=cinema_hall_id)
+    return MovieSession.objects.create(
+        movie=movie,
+        cinema_hall=cinema_hall,
```

Comment:
here you can pass ids directly instead of creating objects first - use cinema_hall_id